src/buttons.js

```diff
@@ -0,0 +1,52 @@
+class Task {
+  static getTasks() {
+    const tasks = localStorage.getItem('tasks');
+    return tasks ? JSON.parse(tasks) : [];
+  }
+
+  static setTasks(tasks) {
+    localStorage.setItem('tasks', JSON.stringify(tasks));
+  }
+
+  static addTask(description) {
+    const tasks = Task.getTasks();
+    const task = {
+      id: tasks.length,
+      description,
+      completed: false,
```

Comment:
- [ ] Great job setting the completed status to false by default. According to the project requirements, you are expected to set the value of the index(id) to the new length of the array after the element is added. So if you are adding the first element, the array length is 0 and the id of the element should be 1 not 0. You can just add `+ 1` on line 14.

Reply:
@Kamba56 Thank you so much for your feedback, but +1 it's not working, so can you help me with the issue, please

Reply:
> @Kamba56 Thank you so much for your feedback, but +1 it's not working, so can you help me with the issue, please

I think if you do `id: tasks.length + 1`. It will work.
I didn't mean you should change the entire line to +1 please